build.js

```diff
@@ -0,0 +1,220 @@
+import { createHash } from 'crypto';
+import { readFileSync, writeFileSync } from 'fs';
+import less from 'less';
+import { extname } from 'path';
+import { minify } from 'uglify-js';
+
+import { CONTENT_PATH_PREFIX, getAllFiles, PAGES_INDEX_PATH, LESS_LOCATIONS, parseFrontMatter, writeFileSyncInFolder } from './common.js';
+
+if (!process.env.hasOwnProperty('GITHUB_USERNAME_TOKEN')) {
+  console.error('GITHUB_USERNAME_TOKEN env variable is not set. Format is: username@token');
+  process.exit(1);
+}
```

Comment:
This change breaks just running `docker build` without a token, which is quite useful for me (and also the reason why GH action is failing).

Reply:
Ahhh, I first thought you welcomed the change to be reminded to set the token locally... Now you will just get a log message, as I assume the token will be set in CI... 👌